samples/csharp_dotnetcore/50.teams-messaging-extensions-search/TeamsAppManifest/manifest.json

```diff
@@ -45,5 +49,8 @@
         }
       ]
     }
+  ],
+  "validDomains": [
+    "teamsmessagingextensionssearch20210304124622.azurewebsites.net"
```

Comment:
Please change this to a placeholder string, and add something to the readme which indicates this should be updated to the customer's deployed bot's url. (similar to the 54.teams-task-module sample)